### PR TITLE
playwright test importing from url and deleting dictionary

### PR DESCRIPTION
**Pull request by StefanVukovic99**

### CODE REVIEW

File: test/playwright/visual.spec.js [hunk 1]
```diff
@@ -48,6 +48,20 @@ test('visual', async ({page, extensionId}) => {
...
     await page.locator('input#advanced-checkbox').evaluate((/** @type {HTMLInputElement} */ element) => element.click());
 
+    // Import jmdict_swedish.zip from a URL
+    await page.locator('.settings-item[data-modal-action="show,dictionaries"]').click();
+    await page.locator('button[id="dictionary-import-button"]').click();
+    await page.locator('textarea[id="dictionary-import-url-text"]').fill('https://github.com/themoeway/yomitan/raw/dictionaries/jmdict_swedish.zip');
+    await page.locator('button[id="dictionary-import-url-button"]').click();
+    await expect(page.locator('id=dictionaries')).toHaveText('Dictionaries (2 installed, 2 enabled)', {timeout: 5 * 60 * 1000});
+
+    // Delete the jmdict_sweedish dictionary
```
Comment (cashewnuttynuts) [inline test/playwright/visual.spec.js:58]: typo: sweedish

**Comment by StefanVukovic99** [comment]
The visual difference is the settings page being 7px shorter for some reason. It may have to do with opening and closing the dictionaries modal.

File: test/playwright/visual.spec.js [hunk 2]
```diff
@@ -48,6 +48,20 @@ test('visual', async ({page, extensionId}) => {
     // Wait for the advanced settings to be visible
     await page.locator('input#advanced-checkbox').evaluate((/** @type {HTMLInputElement} */ element) => element.click());
 
+    // Import jmdict_swedish.zip from a URL
+    await page.locator('.settings-item[data-modal-action="show,dictionaries"]').click();
```
Comment (cashewnuttynuts) [inline test/playwright/visual.spec.js:52]: is there supposed to be a comma? "show,dictionaries"

Comment (StefanVukovic99) [reply test/playwright/visual.spec.js:52]: Yeah it's like that

**Comment by cashewnuttynuts** [comment]
> The visual difference is the settings page being 7px shorter for some reason. It may have to do with opening and closing the dictionaries modal.

I ran the tests locally and these are the results:
settings-fresh(settings-fresh)
settings-actual(settings-actual)
huh? what caused the change here?

**Comment by StefanVukovic99** [comment]
I don't think the 7px thing is worth getting into, so if anyone wants to wave this along I think its fine. 

On your local the dictionaries and general section are ordered differently so probably something to do with old screenshots.

**Review by jamesmaa — APPROVED** [review]

**Review by MarvNC — APPROVED** [review]

**Comment by cashewnuttynuts** [comment]
> On your local the dictionaries and general section are ordered differently so probably something to do with old screenshots. 

Makes sense

**Pull request merged**